documentation/modules/appendix/pages/best-practices.adoc

```diff
@@ -12,56 +12,854 @@ SPDX-License-Identifier: MPL-2.0
+* <<no-the>>
+* <<triple-quotes>>
+* <<keep-indentation>>
+* <<new-line>>
+* <<markdown-where-needed>>
+* <<lists-markdown>>
+* <<notes-markdown>>
+* <<examples-markdown>>
+* <<sources-markdown>>
+* <<markdown-blockquote-summary>>
+
+In addition to that, there are also best practices for:
+
+* <<the-name-attribute>>
+* <<the-preferredname-attribute>>
+
+[[capitalization-start-uppercase]]
+=== Capitalization: start uppercase
+
+A `description` consists of at least a few words forming a phrase or a sentence fragment.
+
+Start with an uppercase letter.
+
+For example:
+
+|===
+|{nok-small} DON'T |{ok-small} DO
+
+|flag indicating whether the asset is currently moving
+|Flag indicating whether the asset is currently moving.
+|represents a single position in space with optional z coordinate
+|Represents a single position in space with optional z coordinate.
+|Signifies the Vertical Distance in Reference to a Historic Mean Sea Level as a Vertical Datum
+|Signifies the vertical distance in reference to a historic mean sea level as a vertical datum.
+|===
+
+[[punctuation-end-with-period]]
+=== Punctuation: end with a period
+
+End your `description` with a period (full stop).
+
+Even though it is just a sentence fragment.
+
+|===
+|Example model element |{nok-small} DON'T |{ok-small} DO
+
+//SOURCE: locations.ttl
+|Property `houseNumber`
+|Number of a building in a street##&nbsp;##
+|Number of a building in a street.
+|===
+
+[[comprehensible-concise]]
+=== Writing style: comprehensible, concise
+
+Apply a neutral writing style that is *easy to understand*.
+This fosters the usability and reuseability of an Aspect Model.
+
+Also, be *concise*.
+To get an idea, think of definitions in an IEC/ISO glossary.
+
+[[no-repeated-preferredname]]
+=== Meaningful: do not just repeat the `preferredName`
+
+Write a meaningful Description.
+It can consist of only a sentence fragment, or it may be longer—but do not just repeat the Preferred Name for the Description.
+
+|===
+|Example model element |{nok-small} DON'T |{ok-small} DO
+
+//SOURCE: plant.ttl
+|Property `productFamily`
+|Product family.
+|Set of products sharing similar features.
+|===
+
+Regarding meaningfulness, see also the details given for the `name` attribute at <<meaningful-names>>.
+
+[[abbreviations-in-descriptions]]
+=== Unmistakable abbreviations
+
+Use abbreviations sparingly.
+
+Only use very common standard abbreviations; ensure they are not misleading.
+See also the details provided for the `name` attribute: <<unmistakable-abbreviations>>.
+
+It is acceptable to use editorial abbreviations such as `e.g.` ("for example") or `i.e.` ("that is") or `etc.` ("and so forth").
+
+[[no-brand-names]]
+=== No brand names
+
+Where possible, avoid brand, product, or company names in Descriptions.
+This fosters reuseability of shared Aspect Models.
+
+[[no-entity-property]]
+=== No "Entity", "Property", etc.
+
+SAMM model element terms like Entity, Property, Aspect, etc. do not belong in the Description.
```

Comment:
I assume it is similar to the names with Suffix. If the name is BlaCharacteristics then perhapts also the description might be "Characteristic of the Property "bla""?